src/tests/system/docs/config.rst

```diff
@@ -0,0 +1,338 @@
+Multihost configuration
+#######################
+
+The multihost configuration file contains definition of the domains, hosts and
+their roles that are available to run the tests. It uses the `YAML
+<https://en.wikipedia.org/wiki/YAML>`__ language.
+
+Basic definition
+****************
+
+.. code-block:: yaml
+
+    domains:
+    - type: <domain type>
+      hosts:
+      - hostname: <resolvable dns host name>
```

Comment:
hostname is not necessary a resolvable one, it is the internal one
There is resolvable external_hostname and ip to acrtually reach the machine form outside.

Reply:
external_hostname and ip were removed. So yes, hostname must be resolvable.

Reply:
Essentially, hostname is was used to be external_hostname.

Reply:
This is unexpected and unwanted limitation.
To run in the idm-ci we do not want to mess with local name resolution.
When I change local dns to resolve "client" to some IP I loose the option to run multiple suites in parallel from multiple working directories against multiple set of machines which is what is exactly done in jenkins and sometiems even when teting locally somthing from LTE.

We need to be able to specify the ip addresses and possibly an external hostname.

Reply:
ssh.host|user|password was added to set the IP or hostname for ssh connection.